### PR TITLE
Add release note entries for #16495, #16496, #16497

### PR DESCRIPTION
**Pull request by jeeminso**

### CODE REVIEW

File: docs/appendices/release-notes/5.8.2.rst [hunk 1]
```diff
@@ -65,3 +65,6 @@ Fixes
 - Fixed an issue that caused ``WHERE`` clause to fail to filter rows when the
   clause contained :ref:`ANY <sql_any_array_comparison>` over array literals
   that are empty or containing only nulls.
+
+- Fixed an issue that caused ``WHERE`` clause containing nested arrays under
```
Comment (matriv) [inline docs/appendices/release-notes/5.8.2.rst:69]:
```suggestion
- Fixed an issue that caused queries with nested arrays, compared with `=` and `= ANY` operators in the ``WHERE`` clause to throw ``ClassCastException```

**Review by matriv — APPROVED** [review]
left a rephrasing suggestion.

File: docs/appendices/release-notes/5.8.2.rst (hunk 1 above)
Comment (jeeminso) [reply docs/appendices/release-notes/5.8.2.rst]: Thanks!

**Comment by jeeminso** [comment]
retest this please

**Pull request merged**